x-pack/plugins/cloud/server/config.test.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { config } from './config';
+
+describe('xpack.cloud config', () => {
+  describe('full_story', () => {
+    it('allows orgId when enabled: false', () => {
```

Comment:
orgId -> org_id

Reply:
Auto merge got this, will fix on next PR.